.github/workflows/docker-update.yml

```diff
@@ -14,8 +14,26 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-      - name: Push to Docker Hub
+      - name: Push to Docker Hub latest
         uses: docker/build-push-action@v2
         with:
           push: true
           tags: profiq/node-java-chrome-ff:latest
+      - name: Get latest chrome version
+        run: |
+          echo ::set-env name=CHROME_VERSION::$(google-chrome --version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1 | sed '/Google Chrome /s///')
+      - name: Push to Docker Hub with chrome version
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: profiq/node-java-chrome-ff:${{ env.CHROME_VERSION }}
+
+  test:
+    name: Run tests
```

Comment:
Btw we should run the tests before we do push. Because with this we are able to tests that we just released something which doesn't work.

The steps should be as follows:
1. Build the image 
2. Test that it works
3. Push it 

But it might be hard to do simply ... between the jobs... 

https://github.com/docker/build-push-action

Btw let's make MR just for the tags and leave the tests for another MR.